apps/condo/domains/banking/access/BankIntegrationOrganizationContext.js

```diff
@@ -0,0 +1,64 @@
+// @ts-nocheck
+/**
+ * Generated by `createschema banking.BankIntegrationOrganizationContext 'integration:Relationship:BankIntegration:CASCADE; organization:Relationship:Organization:CASCADE;'`
+ */
+
+const get = require('lodash/get')
+
+const { throwAuthenticationError } = require('@open-condo/keystone/apolloErrorFormatter')
+const { getById } = require('@open-condo/keystone/schema')
+
+const { checkPermissionInUserOrganizationOrRelatedOrganization } = require('@condo/domains/organization/utils/accessSchema')
+const { queryOrganizationEmployeeFor, queryOrganizationEmployeeFromRelatedOrganizationFor } = require('@condo/domains/organization/utils/accessSchema')
+const { SERVICE } = require('@condo/domains/user/constants/common')
+
+async function canReadBankIntegrationOrganizationContexts (args) {
+    const { authentication: { item: user }, originalInput, context } = args
+    if (!user) return throwAuthenticationError()
+    if (user.deletedAt) return false
+    if (user.isAdmin || user.isSupport) return true
+
+    if (user.type === SERVICE) {
+        // const bankIntegration = get(originalInput, ['organization', 'connect', 'id'])
+        // TODO(VKislov): need to check access for service user with checkBankIntegrationsAccessRights(context, user.id, [bankIntegration])
```

Comment:
this pr?

Reply:
No sir. Need code here from https://github.com/open-condo-software/condo/pull/2943. Added TODO not to forget